src/gd/BraceEnabler.cxx

```diff
@@ -0,0 +1,67 @@
+/*
+ *  BraceEnabler.cxx
+ *  Copyright 2021 ItJustWorksTM
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ */
+
+#include <cstddef>
+#include <core/Godot.hpp>
+#include <gen/Node.hpp>
+#include <gen/__icalls.hpp>
+
+#include "gd/BraceEnabler.hxx"
+
+namespace godot {
+
+void BraceEnabler::_register_methods() { register_method("_notification", &BraceEnabler::_notification); }
+
+void BraceEnabler::_init() {}
```

Comment:
Ideally should check that the Godot version is 3.x, else we risk segfaulting, or at least causing memory corruption

Reply:
what other version would you be expecting lol